### PR TITLE
pom update for release 2.2.11

### PR DESCRIPTION
**Pull request by jacobdotcosta**
ENTSBT-815 ENTSBT-446

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,25 +124,25 @@
 
   <properties>
     <amqp-10-starter.version>2.2.2</amqp-10-starter.version>
-    <dekorate.version>1.0.0</dekorate.version>
+    <dekorate.version>1.0.2</dekorate.version>
     <infinispan-starter.version>2.2.3.Final</infinispan-starter.version>
-    <keycloak.version>9.0.3</keycloak.version>
-    <narayana-starter.version>2.3.2</narayana-starter.version>
+    <keycloak.version>9.0.9</keycloak.version>
```
Comment (aureamunoz) [inline pom.xml:129]: From the jira ticket we are waiting for the upstream version for keycloak but this one is aligned with the downstream one

File: pom.xml [hunk 2]
```diff
@@ -124,25 +124,25 @@
...
 
     <!-- Spring Ecosystem -->
-    <spring-boot.version>2.2.11.RELEASE</spring-boot.version>
-    <spring-cloud-kubernetes.version>1.1.2.RELEASE</spring-cloud-kubernetes.version>
+    <spring-boot.version>2.2.11.RELEASE</spring-boot.version>    
+    <spring-cloud-kubernetes.version>1.1.7.RELEASE</spring-cloud-kubernetes.version>
 
     <!-- Overriden from Spring Boot -->
-    <hibernate.version>5.4.22.Final</hibernate.version>
+    <hibernate.version>5.3.20.Final</hibernate.version>
```
Comment (aureamunoz) [inline pom.xml:141]: hibernate.version downgrading???

Comment (metacosm) [inline pom.xml:141]: I think we should probably keep the more up to date version in the community version.

File: pom.xml (hunk 1 above)
Comment (jacobdotcosta) [reply pom.xml:129]: You're right, although I asked for it I guess I updated before having the answer.

File: pom.xml (hunk 2 above)
Comment (jacobdotcosta) [reply pom.xml:141]: You're right, I ended up applying the same version that was used in Wildfly and didn't notice it was a downgrade.

File: pom.xml [hunk 3]
```diff
@@ -124,25 +124,25 @@
 
   <properties>
     <amqp-10-starter.version>2.2.2</amqp-10-starter.version>
-    <dekorate.version>1.0.0</dekorate.version>
+    <dekorate.version>1.0.2</dekorate.version>
     <infinispan-starter.version>2.2.3.Final</infinispan-starter.version>
```
[the remark this answers is not in the document]
Comment (jacobdotcosta) [reply pom.xml:128]: We don't have the infinispan starter upstream version, I forgot to ask for that.

File: pom.xml (hunk 2 above)
Comment (jacobdotcosta) [reply pom.xml:141]: Ok, I ended up applying what was used in Wildfly without noticing it was a downgrade.

Comment (jacobdotcosta) [reply pom.xml]: Fixed.

Comment (jacobdotcosta) [reply pom.xml]: Fixed.

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jacobdotcosta) [reply pom.xml]: We have it now. Fixed.

**Review by gytis — APPROVED** [review]
Vert.x and Narayana are correct

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (metacosm) [reply pom.xml]: Note that I wasn't saying we should downgrade, just wondering where this version is coming from…

[the remark this answers is not in the document]
Comment (jacobdotcosta) [reply pom.xml]: This isn't just a downgrade. I've checked the infinispan configuration for 2.2.6 to see where did it came from and didn't see any reference from JIRA so I'm not sure what's the origin anymore. This is why I established the one commented oi the 2.2.11 JIRA.

[the remark this answers is not in the document]
Comment (jacobdotcosta) [reply pom.xml:128]: It comes from 2.2.10, not 2.2.6, so I got confused. The configuration has been reverted to version `2.2.3.Final`.

**Pull request merged**